code/io/keycontrol.cpp

```diff
@@ -2805,6 +2816,17 @@ int button_function(int n)
 			hud_escort_target_next();
 			break;
 
+		case TOGGLE_AUTO_SHIELD_EQUALIZE:
+			Players[Player_num].flags ^= PLAYER_FLAGS_AUTO_SHIELD_EQUALIZE;
+			if (Players[Player_num].flags & PLAYER_FLAGS_AUTO_SHIELD_EQUALIZE) {
+				Players[Player_num].flags &= ~PLAYER_FLAGS_AUTO_SHIELD_EQUALIZE_OVERRIDE;
+				button_function(SHIELD_EQUALIZE);
+				HUD_printf(XSTR("Auto shield equalization activated", 1638));
```

Comment:
Since this uses a `printf` style format string, the format string should not be supplied from a function call. This should use `"%s"` as the format string and the `XSTR` call should be the format parameter.